src/shogun/lib/SGVector.cpp

```diff
@@ -338,15 +338,6 @@ SGVector<T> SGVector<T>::slice(index_t l, index_t h) const
 	return SGVector<T>(vector, h - l, l);
 }
 
-template<class T>
-void SGVector<T>::add(const T x)
-{
-	assert_on_cpu();
-	require(vector, "Addition possible for only non-null vectors.");
-	for (int32_t i=0; i<vlen; i++)
-		vector[i]+=x;
-}
-
 template<class T>
 void SGVector<T>::add(const SGSparseVector<T>& x)
```

Comment:
@gf712 @karlnapf Since there is no specialization for SGSparseVector in linalg, What should I replace these calls with? 
Or can you guide me as to how I can write the add function for Sparse Vectors and add it to linalg?

Reply:
I think for now you can leave this

Reply:
is it used anywhere?